testsuite/cmake/test_doxygen.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
since we decided to not test sphinx nor doxygen, we can remove the testscripts.

Reply:
yes